src/types/transaction.rs

```diff
@@ -362,8 +363,14 @@ pub struct FundingResult {
 mod tests {
     use super::*;
     use bitcoin::blockdata::script::Script;
+    use bitcoin::util::hash::Sha256dHash;
```

Comment:
Since ` pub type TransactionId = Sha256dHash;` is defined above, can't you use `TransactionId` here instead? 
See below in the test: `Sha256dHash::from_hex...` .

Reply:
Yep! Good find!